Source/Charts/Highlight/PieHighlighter.swift

```diff
@@ -17,12 +17,10 @@ open class PieHighlighter: PieRadarHighlighter
 {
     open override func closestHighlight(index: Int, x: CGFloat, y: CGFloat) -> Highlight?
     {
-        guard let set = chart?.data?.dataSets[0]
+        guard let set = chart?.data?.dataSets[0],
```

Comment:
Combining these guard statements makes it harder to read. I think it was easier to understand before

Reply:
What about all the other places in the framework where we are writing `guard` statements like this? Although usually it's written like 
```
guard
    let set = chart?.data?.dataSets[0],
    let entry = set.entryForIndex(index)
    else { return nil }
```